parser/include/parser/parsercontext.h

```diff
@@ -32,10 +32,10 @@ enum class IncrementalStatus
 struct ParserContext 
 {  
   ParserContext(
-      std::shared_ptr<odb::database> db_,
-      SourceManager& srcMgr_,
-      std::string& compassRoot_,
-      po::variables_map& options);
+    std::shared_ptr<odb::database> db_,
+    SourceManager& srcMgr_,
+    std::string& compassRoot_,
+    po::variables_map& options);
```

Comment:
There is also a naming issue for argument `options` which could be fixed now, it should be `options_`, according to the coding convention.